src/js/records/entities/balance.record.js

```diff
@@ -1,14 +1,24 @@
 import { AssetRecord } from './asset.record'
 import safeGet from 'lodash/get'
+import config from '@/config'
 
 export class BalanceRecord {
-  constructor (record = {}) {
+  constructor (record = {}, precision) {
     this._record = record
 
     this.id = record.id
     this.asset = new AssetRecord(record.asset)
 
-    this.balance = safeGet(record, 'state.available')
+    this.balance =
+      this.balanceToPrecision(safeGet(record, 'state.available'), precision)
+
     this.convertedBalance = record.convertedBalance
   }
+
+  balanceToPrecision (balance, precision = 0) {
```

Comment:
What will happen if the token will have trailingDigits = 0?

Reply:
method `balanceToPrecision` return integer number without trailing digits

Reply:
no